src/components/camera.js

```diff
@@ -128,8 +153,8 @@ module.exports.Component = registerComponent('camera', {
     var camera = this.camera;
     var system = this.system;
     if (!oldData || oldData.userHeight !== data.userHeight) {
-      this.removeHeightOffset();
-      this.addHeightOffset();
+      this.userHeightOffset = oldData.userHeight || 0;
```

Comment:
OK, so this addresses if you update `userHeight` while in VR? That'd be an odd case, but we can maybe clean up later.